ZenPacks/zenoss/WBEM/patches.py

```diff
@@ -44,7 +46,11 @@ def parseErrorAndResponse(self, data):
 
         error = xml.find('.//ERROR')
 
-        if error is None:
+        if error is not None:
+            msg = error.get('DESCRIPTION')
+            if msg and "context cannot be found" in msg:
+                error.set('DESCRIPTION', "Response is not complete for {} classname".format(self.classname))
```

Comment:
Also, the line is longer than 79 characters.

Reply:
79 is a legacy senseless constraints, originally this module uses 120 symbols limit.